### PR TITLE
Micromouse Team Page Finished

### PR DESCRIPTION
**Pull request by AshwinSant5**
Added in member cards details into "src/Assets/Lists/members.js" and images into "src/Assets/Images/team/2024-2025". Each member card should containing images, names, positions and socials.

### CODE REVIEW

**Comment by carmen-chau** [comment]
Hey! I'll leave the further review to @li-ruihao, but one thing to point out: Please uncommit the files `package-lock.json` and `yarn.lock`. Thanks :)

File: src/Assets/Lists/members.js [hunk 1]
```diff
@@ -200,7 +200,23 @@ const memberData = {
         },
         "Micromouse": {
             membersList: [
-                
+                {
+                    fullName: "Aahan Patel",
+                    position: "Project/Subteam Director",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:205]: Please change this to **Micromouse Director**

File: src/Assets/Lists/members.js [hunk 2]
```diff
@@ -200,7 +200,23 @@ const memberData = {
...
+                    emailLink: "aahan.patel@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/Aahanp",
+                },
+                {
+                    fullName: "Aidan Principe",
+                    position: "Subsystems Lead",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:211]: Please change this to **Mechanical Lead**

File: src/Assets/Lists/members.js [hunk 3]
```diff
@@ -200,7 +200,23 @@ const memberData = {
...
+                    emailLink: "Aidan.principe@mail.utoronto.ca",
+                },
+                {
+                    fullName: "Daniel Neagu",
+                    position: "Subsystems Lead",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:216]: Plesae change this to **Electrical Lead**

**Review by li-ruihao — CHANGES_REQUESTED** [review]

**Pull request closed without merging**